docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -109,16 +109,57 @@ Follow the instructions below based on the query type:
+   * **Target**. The value in percentage you want to target for the SLO (for example, you'd enter `99` for 99%).
+   * **Compliance Type**. **Rolling** provides a sequence of recent days for the Compliance Period, such as the last 7d or last 30d. **Calendar** calculates over a window of time for a Week.
+   * **Timezone**. Selecting a timezone is important to accurately assign events on the boundary of a compliance period, such as events received at 11:59 PM in a particular time zone.<br/><img src={useBaseUrl('img/observability/slo-create-slo.png')} alt="Reliability Management SLO SLI" width="400"/>
+9. Enter SLO Details:
+   * **Name**. Name of the SLO.
+   * **Description**. Short explanation of your SLO.
+   * **Tags**. Tags are key/value pairs which associate SLO with a particular category. [Learn more here](#slo-tags-and-filters). These tags have additional metadata to describe an SLO beyond the name and description, which can help you categorize, search, and filter your SLOs.<br/>Select the key and its possible values from the tags dropdown, or create your own key/value pair for the tag.<br/>
+  :::info
+  You can associate multiple tags with your SLO.
+  :::
+  <img src={useBaseUrl('img/observability/slo-details.png')} alt="SLO Details" width="800"/>
 10. Click **Save**. To create a monitor, click [Save and Create Monitor](#create-an-slo-monitor).
 
 
+### SLO Tags and Filters
+
+You can add key/value pair tags to your SLOs to allow you to better organize, sort, and filter them. As an example, you might find it useful to add variables such as environment, application name, and region.
+
+#### Add a Tag
+
+To add a tag(s) to an existing SLO:
+
+1. In Sumo Logic, click **Manage Data** > **Monitoring** > **SLOs** tab.
+1. Click on any SLO line item in your list, then click **Edit**.
+1. Scroll down to section **(3) SLO Details**. Click on **Tags (Optional)** and type in a new tag or select an existing tag.
+1. Click **Save**.
+
+To add a tag while creating a new SLO:
+
+1. In Sumo Logic, click **Manage Data** > **Monitoring** > **SLOs** tab.
+1. Click **Add** > **New SLO**.
+1. After you've filled out sections **1** and **2**, scroll down to section **(3) SLO Details**. Click on **Tags (Optional)** and type in a new tag or select an existing tag.
+1. Click **Save**.
+
+#### Filter Tags
+
+After you've added a tag, you'll see it populate in the **Tags** column next to your SLO in the list.
+
+To filter your SLOs by tag:
+
+1. Click **Add a filter** at the top of the screen, then click **Tag**.<br/><img src={useBaseUrl('img/observability/slo-tags.png')} alt="slo-tags.png" />
+1. Scroll through the list of tags or type in the tag name you're looking for.<br/><img src={useBaseUrl('img/observability/slo-tags.gif')} alt="slo-tags.gif" />
+
+#### Use cases
+
+You can leverage SLO tags in **Log Search** queries. In this example, we run a query to display all SLOs. You'll see a **Tags** column here.
```

Comment:
did you intend to include an SLO lookup query here i.e. `cat sumo;//content/slos` and an associated image?

Reply:
good catch!